rs/bitcoin/checker/tests/tests.rs

```diff
@@ -118,6 +120,18 @@ fn decode<'a, T: CandidType + Deserialize<'a>>(result: &'a WasmResult) -> T {
     }
 }
 
+#[test]
+fn test_get_tx_cycle_cost() {
+    assert_eq!(
+        get_tx_cycle_cost(INITIAL_MAX_RESPONSE_BYTES, 13),
+        97_063_200
+    );
+    assert_eq!(
+        get_tx_cycle_cost(INITIAL_MAX_RESPONSE_BYTES, 34),
+        296_697_600
```

Comment:
understanding question: is there some security margin there? The amount for the 13 nodes subnet exactly matches the amount in Manu's error message but the one for the 34 node subnet doesn't (was `but 284_580_000 cycles are required.`). I think it makes sense to have some margin, just wandering if it's on purpose?

Reply:
I think the margin comes from the input size. The cost calculation assumes input is 1024 byte, but actual input could be less.